### DISCUSSION TITLE
Actividad 4 Grupo 8

### DISCUSSION THREAD

**Discussion by tomaszbk** [frlp-utn-ingsoft/catedra#154] [Show and tell]
https://github.com/tomaszbk/vetsoft

**Comment by ivanduva** [comment]
Buenas! Les puedo pedir el link a donde lo deployaron?

**Reply by tomaszbk** [reply]
Buenas, si: https://vetsoft-grupo-8.onrender.com/

**Reply by ivanduva** [reply]
Bueno, ahí revisamos la Actividad.
Están redondeando un muy buen trabajo integrador. Sólo les vamos a pedir algunos cambios menores para cerrarlo:

- En cuanto a la validación de que el teléfono empiece con 54, el test e2e no testea nada, no hay ningún expect. Arreglen eso y estaría bueno que metan uno de integración.
- En cuanto a la validación del email, se permite agregar "@vetsoft.com" sin nada adelante. Arreglen eso. También estaría bueno que agreguen uno e2e.
- En cuanto a la validación de que el teléfono sea numérico, no vemos ningún test. Solo modificaron los anteriores. Agreguen al menos uno. Idealmente hagan uno de cada una de las estrategias (unidad, integración y e2e). También fijense que dejaron un typo en la línea 41 del form, debería ser `type="number"`. Igual esto último no importa tanto porque lo chequean de otra manera.
- También estaría bueno que metan un e2e sobre la validación del nombre.
- Luego de hacer todo, saquen una release.

Recuerden que los cambios deben ser realizados por las personas que se encargaron de cada issue.

Después en cuanto a detalles del procedimiento, solo un comentario: hay algunos PRs que los mergearon con la CI fallando, incluso sabiendo y comentando que lo corregían en otro PR. Esto no es una buena práctica, todos los PRs deberían tener la CI pasando.
Más allá de esto, vemos un muy buen desempeño en los PRs y las reviews. Vimos incluso que agregaron un template para documentarlos.

Arreglando eso ya tendrían el trabajo integrador completamente aprobado.

**La última fecha límite de será el último día de cursada antes de las vacaciones: Lunes 8 de Julio 23:59 hs.**

Saludos

**Reply by tomaszbk** [reply]
Buenos dias! ya hicimos los cambios y lanzamos una nueva release

![image](https://github.com/frlp-utn-ingsoft/catedra/assets/85851175/98c2036b-c843-4966-9738-4651dad261b4)

Saludos

**Comment by ivanduva** [comment]
Buenas chicos,
El trabajo integrador queda aprobado.
Felicitaciones por el buen trabajo.
Saludos!
